rules-tests/Php82/Rector/New_/FilesystemIteratorSkipDots/Fixture/constant_present_with_fqn.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
please rename fixture file name to : `skip_constant_present_with_fqn.php.inc`

Reply:
I renamed the file